test/test_document.rb

```diff
@@ -126,6 +128,24 @@ def test_with_default_entity
             doc.root.children.first.value
           end
         end
+
+        def test_entity_expansion_text_limit
+          xml = <<-XML
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE member [
+  <!ENTITY a "&b;&b;&b;">
+  <!ENTITY b "&c;&d;&e;">
+  <!ENTITY c "xxxxxxxxxx">
+  <!ENTITY d "yyyyyyyyyy">
+  <!ENTITY e "zzzzzzzzzz">
+]>
+<member>&a;</member>
+          XML
+
+          REXML::Security.entity_expansion_text_limit = 90
+          doc = REXML::Document.new(xml)
+          doc.root.children.first.value
```

Comment:
Can we use `assert_equal` here?

```suggestion
          assert_equal(XXX, doc.root.children.first.value.bytesize)
```

Reply:
OK, I see.